### DISCUSSION TITLE
Any way to disable analytics globally?

### DISCUSSION THREAD

**Discussion by 0xMySt1c** [oven-sh/bun#2605] [Q&A] [answered]
Is there a way to use DISABLE_BUN_ANALYTICS=1 globally.

Also it would be nice if there was also a way to check current status.
Next has a nice way to deal with this.

```
npx next telemetry status
npx next telemetry enable / disable
```

**Comment by Jarred-Sumner** [comment, marked as answer]
Currently, Bun doesn't actually upload any analytics. I disabled the code for it awhile ago. We will eventually re-enable that though
